### PR TITLE
added suport for the contructor to take in a bignum

### PR DESCRIPTION
**Pull request by wanderer**
with bignum you can do

```
var a = bignum(8)
var b = bignum(a)

console.log(b.toNumber())  // 8

```

this adds support for this behavoir.

### CODE REVIEW

**Comment by weilu** [comment]
Nice! I've updated the dev dependencies. Can you remove your changes on `package.json` and add a test for the supported constructor type you are adding please?

**Comment by wanderer** [comment]
ok updated

**Comment by weilu** [comment]
@wanderer Perhaps I didn't make myself clear. You can remove the `package.json` from this PR as I've already updated that on master: https://github.com/weilu/bigint-browserify/commit/056d65b760065aba7983f6a664063e408e01942e. Aside from the removal, I was asking if you can add a test in this file https://github.com/weilu/bigint-browserify/blob/master/test/index.js to cover the new constructor type you are adding.

**Comment by weilu** [comment]
Also, I'd like to have a clean history. If you could squash your changes into a single commit that'd be great.

**Comment by wanderer** [comment]
ok reverted the package. How do you do the squash thing?

**Comment by weilu** [comment]
`git rebase -i HEAD~6` then use the fixup option.
Read more here: http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History.

I'm going to ask again, could you add tests to the interface you've added? namely the bignum constructor and `toNumber` function. If not, just do the squash I'll add tests to those :)

**Comment by wanderer** [comment]
ok added tests, thanks you @weilu for the new git knowledge.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "jsbn": "0.0.0"
   },
   "devDependencies": {
-    "bignum": "^0.9.1",
-    "tape": "^3.5.0"
+    "tape": "~1.0.4",
+    "bignum": "git+https://github.com/dcousens/node-bignum.git#isbitset"
```
Comment (weilu) [inline package.json]: Why are these dependencies downgraded?

Comment (wanderer) [inline package.json]: fixed now

**Comment by weilu** [comment]
Thank you @wanderer :)

**Pull request merged**